mantidproject/mantid
# Increase information available to the user in the error reporter

This PR adds a more details button to the error reporter, to display the stack trace to the user. Often this can contain useful information, which can help the user diagnose/fix the issue. This PR also adds a configure file to generate a launch script for window builds which will run the error reporter if a crash is experienced. 



**To test:**

1. Make sure you have set the python extensions directory
2. Create a script with an error in e.g:

```
class ExampleAlg(PythonAlgorithm):
    def PyInit(self):
        self.declareProperty('numRuns', 0)
    def category(self):
        return 'Muon'
    def name(self):
        return 'ExampleAlg'
    def PyExec(self):
        n=self.getProperty('numRuns').value
        print("Input number:", n)
```

3. If you are on windows, launch workbench using the launch_workbench.ps1 in the bin/../ directory
4. The error reporter should show up
5. There should be a more details button, which shows your user enviroment and the stacktrace.



Fixes #27462. 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
